src/pages/CoinListPage/CoinListPage.module.scss

```
@@ -1,7 +1,6 @@
-@import "@styles/variables";
+@import "../../styles/variables";
```

в импорте scss перед алиасом надо ставить тильду
```suggestion
@import "~@styles/variables";
```